### ISSUE TITLE
Angled Panels do not have proper clips

### ISSUE THREAD

**Issue by Byzarru** [BEEmod/BEE2-items#1648]
I'm working in Overgrown with a static angled panel, and the voxel it occupies is blocked off from the player. It blocks portal shots, which is especially troublesome in a puzzle I'm working on. This is also an issue in P1, and I assume it is in all other styles.

**Comment by ZombiePwner1** [comment]
Are you able to send a screenshot?

**Comment by TeamSpen210** [comment]
This is intentional - the clips are the same or very similar in all styles, so if you switch it won't break your puzzle.

**Comment by Byzarru** [comment]
It's also very awkward and hinders portal shots in a puzzle I'm making. I'll send a screenshot later tonight.

**Comment by TeamSpen210** [comment]
It should take up a ½ triangle for 90° at most, reducing in size.

**Comment by Byzarru** [comment]
Floating above the arms:
![20170303230520_1](https://cloud.githubusercontent.com/assets/22107242/23576267/9b24186a-0066-11e7-88e5-50ee127eeacb.jpg)
![20170303230530_1](https://cloud.githubusercontent.com/assets/22107242/23576268/9b29155e-0066-11e7-8d94-10b38938c930.jpg)
Can't walk into the arms:
![20170303230535_1](https://cloud.githubusercontent.com/assets/22107242/23576269/9b2a7e08-0066-11e7-926d-49c1d907d704.jpg)
I'm shooting straight ahead. Angled panels needlessly and unintuitively block portal shots.
![20170303230546_1](https://cloud.githubusercontent.com/assets/22107242/23576271/9b2ef6cc-0066-11e7-875c-fa513c1fad46.jpg)
Tried it here, too.
![20170303230555_1](https://cloud.githubusercontent.com/assets/22107242/23576270/9b2ed656-0066-11e7-9114-78f8c9c47d8c.jpg)
Now, I've trapped myself behind the arms. I can't jump (or stop moving, for that matter).
![20170303230618_1](https://cloud.githubusercontent.com/assets/22107242/23576272/9b30bc6e-0066-11e7-8289-1113b17d1734.jpg)
Logic dictates that I should be able to sneak underneath that panel, but an invisible barrier prevents me.
![20170303230630_1](https://cloud.githubusercontent.com/assets/22107242/23576273/9b346a9e-0066-11e7-9004-c3a759554fea.jpg)
And now, for floating cubes!
![20170303230649_1](https://cloud.githubusercontent.com/assets/22107242/23576274/9b3714c4-0066-11e7-9069-fa0317bb366e.jpg)
![20170303230707_1](https://cloud.githubusercontent.com/assets/22107242/23576275/9b392d0e-0066-11e7-94d2-5d968cac6b97.jpg)

**Comment by Byzarru** [comment]
I can tell now that the collisions are angled, but the overall result is an item that becomes unnatural and unrealistic.

**Comment by ZombiePwner1** [comment]
@Byzarru Do not remove the overgrown and portal 1 labels if this bug is to do with overgrown and portal 1 then you need the labels.

**Comment by Byzarru** [comment]
If it affects all styles, I do not need the labels.

**Comment by TeamSpen210** [comment]
2, 3, 4, 6 and 8 happen in the original angled panel item in the same way. The thing is that there might be a map depending on this behavior, I don't want to break maps if you switch styles.

**Comment by Byzarru** [comment]
It's still rather jarring that this clip exists as it seems to defy some of the very design logic that BEE item development has been following (invisible objects are a no-no because they are unrealistic). Add to that 4 and 6 are obvious portal shots, but are not in reality.

**Comment by TeamSpen210** [comment]
No, it's actually all over the place. Check piston platforms, track platforms or angled panels in Old Aperture Style, they do the same thing.

**Comment by Luke18033** [comment]
How about add a stylevar for this?

**Comment by Byzarru** [comment]
That sounds like a good idea.

**Comment by Luke18033** [comment]
Can we just implement this for the current items (no stylevar)? Total style matching isn't really necessary, I highly doubt any good puzzle would be broken by angled panels having slightly different clips.
